src/main/java/com/notably/logic/commands/suggestion/OpenSuggestionCommand.java

```diff
@@ -47,29 +47,31 @@ public void execute(Model model) {
      *
      * @return List of all possible paths
      */
-    private List<AbsolutePath> getPossiblePaths(AbsolutePath path, Model model) {
+    private List<AbsolutePath> getPossiblePaths(List<AbsolutePath> paths, Model model) {
         List<AbsolutePath> possiblePaths = new ArrayList<>();
 
         Queue<AbsolutePath> pathQueue = new LinkedList<>();
-        pathQueue.offer(path);
 
-        while (!pathQueue.isEmpty()) {
-            AbsolutePath currentPath = pathQueue.poll();
+        for (AbsolutePath path : paths) {
+            pathQueue.offer(path);
 
-            List<BlockTreeItem> childrenBlocks = model.getBlockTree().get(currentPath).getBlockChildren();
-            List<AbsolutePath> childrenPaths = childrenBlocks
-                    .stream()
-                    .map(item -> {
-                        List<String> combinedComponents = new ArrayList<>(currentPath.getComponents());
-                        combinedComponents.add(item.getTitle().getText());
-                        return AbsolutePath.fromComponents(combinedComponents);
-                    })
-                    .collect(Collectors.toList());
-            pathQueue.addAll(childrenPaths);
+            while (!pathQueue.isEmpty()) {
+                AbsolutePath currentPath = pathQueue.poll();
 
-            possiblePaths.add(currentPath);
-        }
+                List<BlockTreeItem> childrenBlocks = model.getBlockTree().get(currentPath).getBlockChildren();
+                List<AbsolutePath> childrenPaths = childrenBlocks
+                        .stream()
+                        .map(item -> {
+                            List<String> combinedComponents = new ArrayList<>(currentPath.getComponents());
+                            combinedComponents.add(item.getTitle().getText());
+                            return AbsolutePath.fromComponents(combinedComponents);
+                        })
+                        .collect(Collectors.toList());
+                pathQueue.addAll(childrenPaths);
 
+                possiblePaths.add(currentPath);
+            }
+        }
         return possiblePaths;
```

Comment:
I believe you should sort the result somehow? Otherwise, the suggestions might end up as follows

<img width="1132" alt="Screenshot 2020-04-10 at 3 13 10 PM" src="https://user-images.githubusercontent.com/38042032/78971321-e1383480-7b3d-11ea-82d9-fd6115f4851a.png">

Reply:
Aight!